contracts/ComptrollerG3.sol

```diff
@@ -1246,7 +1246,7 @@ contract ComptrollerG3 is ComptrollerV3Storage, ComptrollerInterface, Comptrolle
      */
     function transferComp(address user, uint userAccrued, uint threshold) internal returns (uint) {
         if (userAccrued >= threshold && userAccrued > 0) {
-            Comp comp = Comp(getCompAddress());
```

Comment:
Can all these old Comptroller contracts be discarded or no?

Reply:
We can do some refactoring after launch. Let's do as few changes as possible for now